src/plots/ternary/ternary.js

```diff
@@ -328,12 +328,12 @@ proto.adjustLayout = function(ternaryLayout, graphSize) {
     _this.layers.bgrid.attr('transform', bTransform);
 
     var aTransform = 'translate(' + (x0 + w / 2) + ',' + y0 +
-        ')rotate(30)translate(0,-' + aaxis._offset + ')';
+        ')rotate(30)translate(0,' + -aaxis._offset + ')';
```

Comment:
When `_offset` is negative we use to get:

![image](https://user-images.githubusercontent.com/6675409/35876084-37fa2c9a-0b3f-11e8-9703-4b62afb321c5.png)

that is double `-` in the translate transform string.

Looks like `nw.js@0.12` doesn't care, so this went unnoticed in our image tests.